Helm/opensearch-dashboards/values.yaml

```diff
@@ -0,0 +1,136 @@
+# Default values for opensearch-dashboards.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
+replicaCount: 1
+
+image: "opensearchproject/opensearch-dashboards"
+imageTag: "latest"
```

Comment:
Lets tie this to the latest version, not this tag. It's dangerous. Long term, we will forget this. Guaranteed.

Reply:
I did not get you. Can you re-iterate? Should I tie it with the `appversion`?

Reply:
I think he means the latest OpenSearch version such as 1.0.0 (docker tag) instead of `latest`.

Reply:
Correct, chart version and app version should probably be in sync and the tag should reference the latest (tested) image. `latest` is dangerous

Reply:
Got it. I will make the change.

Reply:
@TheAlgo Possibly similar to the implementation by @smlx here?
https://github.com/opensearch-project/opensearch-devops/pull/17

Reply:
@peterzhuamazon I have implemented it the similar way and updated the PR already.

Reply:
Thanks.